docs/components/best-practices/development/testing-process-definitions.md

```diff
@@ -124,6 +124,8 @@ In such a test case, you want to test the executable BPMN process definition, pl
 
 In the example above, this is the worker code and the REST API:
 
+<!--- Which REST API are we referring to here? --->
```

Comment:
@berndruecker Can you review the comments I have in the best practices docs to clarify which REST API you are referencing in these?